src/helper/mutations.ts

```diff
@@ -116,8 +116,8 @@ export function useRevisionMutation() {
       }, 200)
       setTimeout(() => {
         NProgress.done()
-        void router.push('/entity/unrevised')
-      }, 3000)
+        void router.push('/entity/unrevised#profile-refresh')
+      }, 4000)
```

Comment:
Why is there a timeout of 4000 seconds? The API cache should be cleared when the mutation is finished and will only be recomputed with the API request.

Reply:
good point. will remove if if works at all :)